lib/std/debug/MemoryAccessor.zig

```diff
@@ -81,9 +81,9 @@ pub fn isValidMemory(address: usize) bool {
     // We are unable to determine validity of memory for freestanding targets
     if (native_os == .freestanding or native_os == .uefi) return true;
 
-    const aligned_address = address & ~@as(usize, @intCast((page_size - 1)));
+    const aligned_address = address & ~(std.heap.pageSize() - 1);
```

Comment:
```suggestion
    const page_size = std.heap.pageSize();
    const aligned_address = address & ~(page_size - 1);
```

then you can reference `page_size` below.